info/mainnet.json

```diff
@@ -4,59 +4,153 @@
     "chainId": 1,
     "rpc": "https://mainnet.infura.io/v3/a4812158fbab4a2aaa849e6f4a6dc605",
     "gateway": "0x4F4495243837681061C4743b74B3eEdf548D56A5",
-    "gasCollector": "0xa57adce1d2fe72949e4308867d894cd7e7de0ef2",
-    "gasReceiver": "0x4154CF6eea0633DD9c4933E76a077fD7E9260738",
-    "constAddressDeployer": "0x617179a15fEAa53Fa82ae80b0fc3E85b7359a748",
+    "constAddressDeployer": "0x98b2920d53612483f91f12ed7754e51b4a77919e",
     "tokenName": "Ether",
     "tokenSymbol": "ETH",
-    "wrappedSymbol": "WETH"
+    "wrappedSymbol": "WETH",
+    "AxelarGasService": {
+      "salt": "AxelarGasService",
+      "address": "0x2d5d7d31F671F86C782533cc367F14109a082712",
+      "implementation": "0xE622B1D44F5061DB10ABa2594F1A8eb2DC1fC1d4",
+      "deployer": "0x6f24A47Fc8AE5441Eb47EFfC3665e70e69Ac3F05",
+      "collector": "0x345662ABfE0E257E23552A434D02Ef274cCb6A90"
+    },
+    "AxelarDepositService": {
```

Comment:
shouldn't we add the address for AxearDepositService as well?

Reply:
mainnet doesn't have a deposit service yet, i'll create another PR once we deploy it there